Add step by step examples for ladd and ledit (#372)
AY2122S1-CS2103T-F13-3/tp: docs/UserGuide.md

You can add multiple pieces of homework to a lesson in TAB.
</div>

Example: 

> You have a student named Alex Yeoh with index number 1. You have a lesson with him every Monday at 11am to 12pm, starting from 20 December 2020, and you want to add it in TAB. The end date is not known, and you are getting paid $35 per hour. For the first lesson, you want to assign assignment 1 as homework. 

Reviewer: ```suggestion
You have a student named Alex Yeoh with index number 1. You have a History lesson with him every Monday at 11am to 12pm, starting from 20 December 2020, and you want to add it in TAB. The end date is not known, and you are getting paid $35 per hour. For the first lesson, you want to assign assignment 1 as homework. 
```

You can add multiple pieces of homework to a lesson in TAB.
</div>

Example: 

> You have a student named Betsy Crowe with index number 7. You have a history lesson with her every Sunday at 11am to 12pm, starting from 20 December 2021, and you want to add it in TAB. The end date is not known, and you are getting paid $35 per hour. For the first lesson, you want to assign assignment 1 as homework. 

Reviewer: ```suggestion
Suppose you have a student named Betsy Crowe with index number 7. You have a history lesson with her every Sunday from 11 am to 12 pm, starting from 20 December 2021, and you want to add it in TAB. The end date is not known, and you are getting paid $35 per hour. For the first lesson, you want to assign assignment 1 as homework. 
```

> You have a student named Betsy Crowe with index number 7. You have a history lesson with her every Sunday at 11am to 12pm, starting from 20 December 2021, and you want to add it in TAB. The end date is not known, and you are getting paid $35 per hour. For the first lesson, you want to assign assignment 1 as homework. 

Reviewer 2: ```suggestion
Suppose you have a student named Betsy Crowe with index number 7. You have a history lesson with her every Sunday at 11am to 12pm, starting from 20 December 2021, and you want to add it in TAB. The end date is not known, and you are getting paid $35 per hour. For the first lesson, you want to assign assignment 1 as homework. 
```

1. Type the command `ladd 7 recurring/ date/20 Dec 2021 time/1100-1200 subject/History rates/35 hw/Assignment 1` into the command box.

> 2. Press <kbd>Enter</kbd> – You should see the lesson being added to the seventh student in the displayed list!

Reviewer 2: ```suggestion
2. Press <kbd>Enter</kbd> and you should see that a lesson has been added to the seventh student in the displayed list!
```

> 2. Press <kbd>Enter</kbd> and you should see that a lesson has been added to the seventh student in the displayed list!

Reviewer 2: I think better to mention name rather than index since index is not shown in the picture
```suggestion
2. Press <kbd>Enter</kbd> and you should see that a lesson has been added to Betsy Crowe in the displayed list!
```

</div>

Example:

> You have a student named Alex Yeoh with index number 1. You mistakenly added the wrong start and end date for his lesson with index number 1. The lesson should start on 2 December 2020 and end on 30 December 2020. Furthermore, he has a school event on the third lesson for this recurring lesson which means the third lesson has to be cancelled.

Reviewer: ```suggestion
You have a student named Alex Yeoh with index number 1. You mistakenly added the wrong start and end date for his lesson with index number 1. The lesson should start on 2 December 2020 and end on 30 December 2020. Furthermore, he has a school event on the third lesson (16 Dec 2021) for this recurring lesson which means the third lesson has to be cancelled.
```

</div>

Example:

> You have a student named Betsy Crowe with index number 7. You mistakenly added the wrong start and end date for her lesson with index number 1. The lesson should start on 20 December 2021 and end on 30 December 2021. Furthermore, she has a school event on the third lesson (16 Dec 2021) for this recurring lesson which means the third lesson has to be cancelled.

Reviewer: ```suggestion
Suppose you have a student named Betsy Crowe with index number 7. You mistakenly added the wrong start and end date for her lesson with index number 1. The lesson should start on 20 December 2021 and end on 30 December 2021. Furthermore, she has a school event on the third lesson (16 Dec 2021) for this recurring lesson which means the third lesson has to be cancelled.
```
